front-end/src/components/main-page/MainPage.test.js

```diff
@@ -69,11 +69,21 @@ describe('MainPage', () => {
         expect(screen.getByLabelText("View Analysis")).toBeDisabled();
     });
 
-    it('should dispatch openAnalysisPage when a valid Project ID is entered and View Analysis button is clicked', async() => {
-        store.dispatch(addAnalyzedTranscript({projectId: "1", transcript: [{question: "a", children: {"b": 100,}}]}));
+    it('should dispatch openAnalysisPage when a valid Project ID is entered and View Analysis button is clicked, merge false', async() => {
+        store.dispatch(addAnalyzedTranscript({projectId: "1mergefalse", transcript: [{question: "a", children: {"b": 100,}}]}));
```

Comment:
+1 on camel, tbh you can probs just do a search for mergefalse and change them all to mergeFalse